tests/python/tank_test/tank_test_base.py

```diff
@@ -808,10 +807,10 @@ def add_production_path(self, path, entity=None):
             # create directories
             os.makedirs(full_path)
         if entity:
-            # add to path cache
-            self.add_to_path_cache(full_path, entity)
             # populate mock sg
             self.add_to_sg_mock_db(entity)
+            # add to path cache
+            self.add_to_path_cache(full_path, entity)
```

Comment:
Bad test data. How can you even have something to the path cache if the entity does not even exist in Mockgun? Once again, the fix I had made in Mockgun broke this line.